subsys/fw_metadata/Kconfig.template.fw_metadata_abi

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
is it convention to leave this as newline?

Reply:
I don't know. I added a license instead :).